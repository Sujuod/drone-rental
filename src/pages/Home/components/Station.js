import React from 'react';

const Station = ({ stationNumebr, onSelect, selected }) => {
    const onStationClick = () => {
        onSelect(stationNumebr);
    }
    return (
        <div onClick={onStationClick} style={selected ? selectedStyle : {}} className='card mt-3 pt-4 pb-4 d-flex align-items-center'>
            <div className="card-body">
                Station  {stationNumebr}
            </div>
        </div>
    );
}

export default Station;

const selectedStyle = {
    backgroundColor: 'rgba(219, 227, 231, 0.83)',
    fontSize: '24px'
}