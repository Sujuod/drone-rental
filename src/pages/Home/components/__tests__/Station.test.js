import Station from '../Station';

function setup() {
  const component = shallow(<Station stationNumebr={1}/>);
  return component;
}

describe('Station component', () => {
  const component = setup();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});