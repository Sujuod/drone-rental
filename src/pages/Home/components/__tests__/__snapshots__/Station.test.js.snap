// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Station component should render correctly 1`] = `
<div
  className="card mt-3 pt-4 pb-4 d-flex align-items-center"
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="card-body"
  >
    Station  
    1
  </div>
</div>
`;
